src/Components/Inquiry/Views/InquiryArtistsInCollection.tsx

```diff
@@ -25,6 +31,10 @@ export const InquiryArtistsInCollection: FC = () => {
   const { trackEvent } = useTracking()
 
   useOnce(() => {
+    submitUpdateMyUserProfile({
```

Comment:
Question, as far as I know only when touching the button I haven't started should update it, not when closing the modal. Can you double-check with product please?

Reply:
Here is my understanding, but I can confirm:

- If the user adds an artist, we need to update it so that they aren't prompted to complete their profile within 30 days.
- If the user clicks "I haven't started...", we need to update it so they aren't prompted to add an artist again within 30 days.
- If the user dismisses the modal, we need to update it so they aren't prompted to add an artist again within 30 days.